code/iaas/logic/src/main/resources/META-INF/cattle/core-process/spring-core-processes-context.xml

```diff
@@ -72,6 +71,15 @@
                                             transitioning="deactivating" done="inactive" />
     <process:process name="agent.reconnect" resourceType="agent" start="active,activating" transitioning="reconnecting" done="active" />
 
+
+    <!-- Credential -->
+    <process:process name="credential.create" resourceType="credential" start="requested" transitioning="registering" done="inactive" delegate="credential.activate" />
```

Comment:
The `<process:defaultProcess />` element has an `exclude` attribute.  I would feel better if you did `exclude="restore"` as it is less error prone than manually defining all of these processes.